release/release.go

```diff
@@ -37,41 +37,33 @@ type OS struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+var (
+	versionSignaturePath = "/proc/version_signature"
```

Comment:
/proc/version_signature is an Ubuntu-ism and so this will only every work with Ubuntu derivatives. There are other ways to introspect the kernel, such as /etc/apparmor.d/cache/.features or /sys/kernel/security/apparmor/features/* that could be used for any distro. Eg, you check for what we need, if the kernel doesn't have it, use force devmode (and log this).